README.md

```diff
@@ -1,110 +1,46 @@
 
-* To access Didact, access the Command Palette (`View->Command Palette`, `Ctrl+Shift+P`, or `F1`) and type **Didact**. Select `Open Didact` and it will open with the default Didact file specified in the Settings.
-* To open a local Didact file in the workspace directly, Right-click on the Didact file (`*.didact.md` or `*.didact.adoc`) and select **Start Didact Tutorial from File** from the context menu.
-
-In addition, you now have the `Didact Tutorials` view. 
-
-![Didact Tutorials View](./images/didact-view-with-popup.png)
-
-If you right-click on a tutorial (such as "Didact Demo"), you see a `Start Didact Tutorial` menu, which opens the Didact window or changes the current Didact window to the selected tutorial. These entries are provided by registering new tutorial name/tutorial uri/category combinations with Didact's tutorial registry.
-
-*NEW!* You can also simply mouse over a registered tutorial and hit the 'play' button (triangle) to launch it in the Didact window.
-
-![One Click Open a Didact Tutorial](./images/didact-tutorial-start-button.gif)
-
-## Didact Settings 
-
-To change the default Didact file, access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Default Url`. This can be any valid URL to a Didact formatted Markdown file, including `http://`, `https://`, or `file://` URLs.
-
-To turn on generic Didact completion notifications (off by default), access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Disable Notifications` checkbox. If unchecked, additional notifications will occur when Didact actions are activated in the IDE.
-
-To have Didact open a window with the default Didact file on startup, access the settings (`File->Preferences->Settings`), type **Didact** and set the `Didact: Open Default Tutorial At Startup` checkbox.
-
-![Didact Settings](./images/settings.jpg)
-
-## Auto-complete for Didact in Markdown files
-
-To help you set up Didact tutorials, we've added a number of auto-completions (accessible by pressing `Ctrl+Space` while editing a Markdown file) to provide most of the basic things you'll need.
-
-![Didact Auto-Completion Demo](./images/didact-uri-completion-demo.gif)
-
-Included completions:
-
-* Insert Validate All button
-* Insert Requirements label
-* Insert link to install required VS Code extension
-* Insert link to create temporary folder as WS root
-* Insert link to start didact from File in Extension folder
-* Start a new didact link (inserts `didact://?` and then offers additional completions
-
-From a `didact://?` start, you also get:
-
-* Start terminal with name
-* Send named terminal some text
-* Send named terminal a Ctrl+C
-* Close terminal with name
-* Non-didact command
-* Check CLI for some returned text
-* Check CLI for some success (no text)
-* Check for required extension
-* Check for root folder in the WS
-* Scaffold project
-* Start Didact from Currently Selected File
```

Comment:
Can you point where in the wiki there is the information about auto-completion for Markdown?

Reply:
considering that all of the auto-completion has just been retooled a bit and I don't know that it's really relevant, I'm not too worried about it. but we can definitely add it to the wiki

Reply:
Moved the docs to https://github.com/redhat-developer/vscode-didact/wiki/Didact-Auto-complete and will update when we release a new version that has the newer autocomplete functionality